src/test/java/com/example/TestUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.example;
+
+import com.example.weatherwaveapi.model.response.openweather.WeatherOpenApiContainer;
+import com.example.weatherwaveapi.model.response.openweather.forecastapi.ForecastDataContainer;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.City;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.SunActivityInfo;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.WeatherMetrics;
+import com.example.weatherwaveapi.model.response.openweather.weatherapi.weather.Weather;
+
+import java.util.List;
+
+public class TestUtil {
```

Comment:
Util classes in Java are required to be final and should have a private constructor.But instead of the private constructor, you could use the annotation @UtilityClass